framework/src/main/java/org/checkerframework/common/value/ValueTransfer.java

```diff
@@ -538,6 +544,31 @@ private Range calculateLengthRangeAddition(Range leftLengths, Range rightLengths
         return leftLengths.plus(rightLengths).intersect(Range.INT_EVERYTHING);
     }
 
+    /**
+     * This method is only used to check for nullable nodes in string concatenations.
```

Comment:
The comment on a method should be about what the method does, not how it is used.